src/kpro_sent_reqs.erl

```diff
@@ -0,0 +1,118 @@
+%%%
+%%%   Copyright (c) 2014-2018, Klarna AB
+%%%
+%%%   Licensed under the Apache License, Version 2.0 (the "License");
+%%%   you may not use this file except in compliance with the License.
+%%%   You may obtain a copy of the License at
+%%%
+%%%       http://www.apache.org/licenses/LICENSE-2.0
+%%%
+%%%   Unless required by applicable law or agreed to in writing, software
+%%%   distributed under the License is distributed on an "AS IS" BASIS,
+%%%   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%%%   See the License for the specific language governing permissions and
+%%%   limitations under the License.
+%%%
+
+%%%=============================================================================
+%%% @doc
+%%% This module manages an opaque of sent-request collection.
```

Comment:
Opaque what?  Either "opaque X (e.g. set) of ...", or in this case just dropping the "of" also works.

Reply:
changed to 'opaque collection of sent-requests'.